CWE/events/disasters.txt

```diff
@@ -1148,3 +1148,26 @@ province_event = {
 		}
 	}
 }
+
+country_event = { 
+	id = 294
+	title = "Vajont Disaster" 
+	desc = "EVT_294_DESC" 
+	picture = "ashgabat_earthquake"
```

Comment:
Please create a event pic for this. My policy is that all historical events should use it’s own unique image.

Reply:
Totally forgot my bad